sql-cli/sql_cli/utils.py

```diff
@@ -9,39 +9,44 @@
 from jinja2.runtime import StrictUndefined
```

Comment:
@feluelle, what if utils was a folder/package and within it, we had the `jinja.py` module? This would avoid `utils` becoming a wild place for other things

Reply:
Sure! Works for me. 👍

Reply:
Done in 71d91c9